setup.cfg

```diff
@@ -13,13 +13,13 @@ license_files =
   LICENSE.txt
 
 [options]
-python_requires = >= 3.7
+python_requires = >= 3.8
 setup_requires =
   setuptools >= 40.6
   pip >= 10
 include_package_data = true
 packages = find:
 install_requires =
-    Django>=3.2, <4.2
-    wagtail>=4.1, <5.0
+    Django>=3.2, <4.3
+    wagtail>=4.1, <5.2
```

Comment:
While i have made some changes here they are duplicates of some config in setup.py and don't seem to be used.

Reply:
Ah okay, yeah, looks good